src/ssl.c

```diff
@@ -6685,6 +6889,12 @@ int wolfSSL_CTX_load_verify_locations_ex(WOLFSSL_CTX* ctx, const char* file,
     }
 
     verify = GET_VERIFY_SETTING_CTX(ctx);
+#ifndef NO_WOLFSSL_CM_VERIFY
+    if (flags & WOLFSSL_LOAD_FLAG_USE_CB) {
+        verify = VERIFY_USE_CB;
```

Comment:
This will override the above "verify" settings, which is not okay. Why is this required at all? Isn't it sufficient to just check "cm-verifyCallback" is set?